### PR TITLE
fix var annotations

### PR DESCRIPTION
**Pull request by jens1o**

### CODE REVIEW

File: src/Node/ElseClauseNode.php [hunk 1]
```diff
@@ -14,6 +14,6 @@ class ElseClauseNode extends Node {
     public $elseKeyword;
     /** @var Token */
     public $colon;
-    /**@var StatementNode | StatementNode[] */
+    /** @var StatementNode | StatementNode[] */
```
Comment (felixfbecker) [inline src/Node/ElseClauseNode.php]: Are spaces between union types allowed? I thought a space separated the type from a description

Comment (jens1o) [inline src/Node/ElseClauseNode.php]: I think it's forbidden, but I don't want to break codestyle to be honest.

Comment (felixfbecker) [inline src/Node/ElseClauseNode.php]: Don't break forbidden code style?

Comment (jens1o) [inline src/Node/ElseClauseNode.php]: Yep, you got it. Until I get a message, I won't break that.

Comment (roblourens) [inline src/Node/ElseClauseNode.php]: It does appear to be wrong, at least it isn't parsed by phpDocumentor when it's space-separated. There are a ton of those in the parser, I'll fix them all at once. Also some in the language server.

**Pull request merged**